src/Avalonia.Input/VectorEventArgs.cs

```diff
@@ -5,6 +5,21 @@ namespace Avalonia.Input
 {
     public class VectorEventArgs : RoutedEventArgs
```

Comment:
Having KeyModifiers on VectorEventArgs feels wrong.
Should there be another type?

Reply:
Why? If we consider VectorEventArgs as one of the types of events with Pointer. (for Thumb he is)

Reply:
Yeah `VectorEventArgs` isn't the right place for this. `VectorEventArgs` is as the name suggests - an `EventArgs` that holds a vector.

We should probably create `class DragDeltaEventArgs : VectorEventArgs` (using the same naming as WPF).

Also instead of placing `KeyModifiers` on `DragDeltaEventArgs`, maybe we should instead expose the original event which caused the event on a property. This can be useful if you want to e.g. find out if the drag was caused by a mouse or a touch interaction.

If so, and we want to port this to 0.10.x, we'll need to create two versions of this PR: one for `master` which changes the types of the event handler to `EventHandler<DragDeltaEventArgs>` and one which doesn't change the signatures for the stable branch: on the stable branch you'll have to do a manual case from `VectorEventArgs` to `DragDeltaEventArgs`